src/Examples.Console/Examples.Console.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net5.0</TargetFramework>
-    <Version>0.2.0-beta</Version>
```

Comment:
So we don't have a version for the example project?

Reply:
I don't think it's needed. It's not published as a package, so I not sure if there's any value in it.